### ISSUE TITLE
Fshare security changes

### ISSUE THREAD

**Issue by bigdargon** [pyload/pyload#3575]
### Describe the problem

Fshare has recently changed its security. When logging in too many times (same ip address), Fshare will notify the status of multiple devices logged and do not allow the login to continue. At that time, you must log in manually to your account, delete all login sessions. So I suggest the following:
- Login only once when pyload is running.
- Or after each login and get the download link, then log out.

And currently, with a free Fshare account only download 5 times/day

### Debug log

```
540	07.04.2020 10:56:55	ERROR	ACCOUNT FshareVn:
541	07.04.2020 10:56:55	ERROR	ACCOUNT FshareVn: Could not login user `myh*******` | Login handshake has failed
542	07.04.2020 10:56:55	INFO	ACCOUNT FshareVn: Updating account info for user `myh*******`...
543	07.04.2020 10:56:55	INFO	ACCOUNT FshareVn: Relogin user `myh*******`...
544	07.04.2020 10:56:57	ERROR	ACCOUNT FshareVn:
545	07.04.2020 10:56:57	ERROR	ACCOUNT FshareVn: Could not login user `myh*******` | Login handshake has failed 
```

### Screenshots

![Screenshot_2020-04-07 Fshare - Bảo mật - Fshare](https://user-images.githubusercontent.com/10969626/78629689-f43ed080-78c1-11ea-9b1f-ef28c7cb61dc.png)

### Additional references

I sent a premium account to email `nitzo2001 at yahoo dot com`

**Comment by GammaC0de** [comment]
They have added random captcha to the login form, even with normal browser:
![Capture](https://user-images.githubusercontent.com/11036551/78720710-d65b8500-792e-11ea-9dc7-ae19a1d0cad1.PNG)

**Comment by bigdargon** [comment]
It's correct. Captcha is random, when first login will have login notification in many devices. But when trying to login again from homepage `fshare.vn` will not have captcha (?)

**Comment by GammaC0de** [comment]
Plugin updated to include captcha support (https://github.com/pyload/pyload/commit/90930a07053a6c874e1f6767728724a88a8aa10b).
I could not test it so please confirm it works.

Please note that you would need to open a `9kw.eu` account and activate and configure the `Captcha9Kw` to use automated captcha solving.

Also, when I login to `fshare.vn` I get this message:
`Hệ thống ghi nhận mật khẩu của Quý khách đang sử dụng không đủ phức tạp để bảo vệ tài khoản. Mật khẩu an toàn có độ dài ít nhất 10 ký tự bao gồm: 1 chữ cái viết hoa (A-Z), 1 chữ cái viết thường (a-z), 1 ký tự số (0-9) và 1 ký tự đặc biệt (!,@,#,&...). Qúy khách vui lòng đổi mật khẩu khác an toàn hơn. `

So maybe changing the password according to these instruction would eliminate the captcha prompt...

**Comment by bigdargon** [comment]
I changed the password and sent it.  It seems the fshare site has many silly errors.  I only log in 2 or 3 times on the same device and the same ip address but still inform me that I have logged in to multiple devices

**Comment by bigdargon** [comment]
I tried with the download link but I got an error at the login stage, even though I entered the correct captcha.  fshare still shows login in many devices, it seems fshare is trying to remove the downloader.

```
09.04.2020 19:36:15 | INFO | Activating Accounts...
-- | -- | --
370 | 09.04.2020 19:36:15 | INFO | ACCOUNT FshareVn: Adding user `myh*******`...
371 | 09.04.2020 19:36:15 | INFO | ACCOUNT FshareVn: Login user `myh*******`...
372 | 09.04.2020 19:36:28 | INFO | CAPTCHA FshareVn: Captcha \| Using OCR to decrypt captcha...
373 | 09.04.2020 19:36:28 | WARNING | CAPTCHA FshareVn: Captcha \| No OCR result
374 | 09.04.2020 19:36:32 | INFO | ADDON Captcha_9kw: 5391 credits left
375 | 09.04.2020 19:36:41 | INFO | CAPTCHA FshareVn: Captcha \| Captcha result: `70`
376 | 09.04.2020 19:36:41 | INFO | CAPTCHA FshareVn: Captcha \| Path deleted: /home/odroid/.pyload/tmp/captcha_image_FshareVn_78870.png
377 | 09.04.2020 19:36:45 | ERROR | ACCOUNT FshareVn: Could not login user `myh*******` \| Login handshake has failed
378 | 09.04.2020 19:36:45 | INFO | Activating Plugins...
379 | 09.04.2020 19:36:45 | INFO | ADDON XFileSharing: Auto-discover new hosters
380 | 09.04.2020 19:36:45 | INFO | ADDON XFileSharing: Auto-discover new crypters
381 | 09.04.2020 19:36:45 | INFO | pyLoad is up and running
382 | 09.04.2020 19:36:45 | INFO | ADDON UpdateManager: pyLoad is up to date!
383 | 09.04.2020 19:36:45 | INFO | ADDON UpdateManager: All plugins are up to date!
384 | 09.04.2020 19:36:47 | INFO | Download starts: [Phim Media] Dong Tay Cung Luoc 20.mkv
385 | 09.04.2020 19:36:47 | WARNING | HOSTER FshareVn[20]: Plugin may be unstable
386 | 09.04.2020 19:36:47 | INFO | HOSTER FshareVn[20]: Grabbing link info...
387 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link name: MV9DRGORKAG7
388 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link size: 1.22 GiB (1309474465 bytes)
389 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link status: starting
390 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Processing url: https://www.fshare.vn/file/MV9DRGORKAG7
391 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Looking for direct download link...
392 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Direct download link not found
393 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Checking for link errors...
394 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: No errors found
395 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Grabbing link info...
396 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link name: MV9DRGORKAG7
397 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link size: 1.22 GiB (1309474465 bytes)
398 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Link status: starting
399 | 09.04.2020 19:36:48 | INFO | HOSTER FshareVn[20]: Processing as free download...
400 | 09.04.2020 19:36:49 | INFO | HOSTER FshareVn[20]: Waiting 1 minutes...
401 | 09.04.2020 19:36:51 | INFO | ADDON ClickNLoad: Proxy listening on 0.0.0.0:9666

```

You can try to load the download after a few links will give an error
Link: https://www.fshare.vn/folder/2VYXKQP31MSV

**Comment by bigdargon** [comment]
I found that with each link, pyload will log in then get the download link.  I configured 3 downloads at the same time, then the system will record 3 devices logged on the same ip address.  So can we log in once at pyload boot, not with multiple login each time download?

**Comment by bigdargon** [comment]
This is the log when pyload runs longer

```

10.04.2020 00:39:03 INFO      Download finished: [Phim Media] Thien Dinh Ky Duyen 10.mkv
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: File saved
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: Checking download...
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: File is OK
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: Checking file (with built-in rules)...
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: Checking file (with custom rules)...
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: Checking for link errors...
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: No errors found
10.04.2020 02:18:50 INFO      HOSTER FshareVn[63]: No errors found
10.04.2020 02:18:50 INFO      Download finished: [Phim Media] Thien Dinh Ky Duyen 12.mkv
10.04.2020 02:18:51 INFO      Download starts: https://www.fshare.vn/file/4MID442LLUAJ
10.04.2020 02:18:51 WARNING   HOSTER FshareVn[64]: Plugin may be unstable
10.04.2020 02:18:51 WARNING   Download failed: https://www.fshare.vn/file/4MID442LLUAJ | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:18:51 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/4MID442LLUAJ (pid:7)
10.04.2020 02:18:51 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:18:53 INFO      Download starts: https://www.fshare.vn/file/T78CQDDGVT66
10.04.2020 02:18:53 WARNING   HOSTER FshareVn[65]: Plugin may be unstable
10.04.2020 02:18:53 WARNING   Download failed: https://www.fshare.vn/file/T78CQDDGVT66 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:18:53 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/T78CQDDGVT66 (pid:7)
10.04.2020 02:18:53 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:18:55 INFO      Download starts: https://www.fshare.vn/file/XN6NJDYG51FQ
10.04.2020 02:18:55 WARNING   HOSTER FshareVn[66]: Plugin may be unstable
10.04.2020 02:18:55 WARNING   Download failed: https://www.fshare.vn/file/XN6NJDYG51FQ | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:18:55 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/XN6NJDYG51FQ (pid:7)
10.04.2020 02:18:55 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:18:57 INFO      Download starts: https://www.fshare.vn/file/OH4TK29VDW82
10.04.2020 02:18:57 WARNING   HOSTER FshareVn[67]: Plugin may be unstable
10.04.2020 02:18:57 WARNING   Download failed: https://www.fshare.vn/file/OH4TK29VDW82 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:18:57 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/OH4TK29VDW82 (pid:7)
10.04.2020 02:18:57 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:18:59 INFO      Download starts: https://www.fshare.vn/file/JGEDB929T3Q5
10.04.2020 02:18:59 WARNING   HOSTER FshareVn[68]: Plugin may be unstable
10.04.2020 02:18:59 WARNING   Download failed: https://www.fshare.vn/file/JGEDB929T3Q5 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:18:59 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/JGEDB929T3Q5 (pid:7)
10.04.2020 02:18:59 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:01 INFO      Download starts: https://www.fshare.vn/file/YA97YXIIPSK6
10.04.2020 02:19:01 WARNING   HOSTER FshareVn[69]: Plugin may be unstable
10.04.2020 02:19:01 WARNING   Download failed: https://www.fshare.vn/file/YA97YXIIPSK6 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:01 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/YA97YXIIPSK6 (pid:7)
10.04.2020 02:19:01 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:03 INFO      Download starts: https://www.fshare.vn/file/NH8CQGQ3ZEP4
10.04.2020 02:19:03 WARNING   HOSTER FshareVn[70]: Plugin may be unstable
10.04.2020 02:19:03 WARNING   Download failed: https://www.fshare.vn/file/NH8CQGQ3ZEP4 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:03 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/NH8CQGQ3ZEP4 (pid:7)
10.04.2020 02:19:03 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:05 INFO      Download starts: https://www.fshare.vn/file/6S69IHNKN7S4
10.04.2020 02:19:05 WARNING   HOSTER FshareVn[71]: Plugin may be unstable
10.04.2020 02:19:05 WARNING   Download failed: https://www.fshare.vn/file/6S69IHNKN7S4 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:05 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/6S69IHNKN7S4 (pid:7)
10.04.2020 02:19:05 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:07 INFO      Download starts: https://www.fshare.vn/file/IS4ZDJALWF3Y
10.04.2020 02:19:07 WARNING   HOSTER FshareVn[73]: Plugin may be unstable
10.04.2020 02:19:07 WARNING   Download failed: https://www.fshare.vn/file/IS4ZDJALWF3Y | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:07 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/IS4ZDJALWF3Y (pid:9)
10.04.2020 02:19:07 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:09 INFO      Download starts: https://www.fshare.vn/file/YLTZ915BT1BA
10.04.2020 02:19:09 WARNING   HOSTER FshareVn[74]: Plugin may be unstable
10.04.2020 02:19:09 WARNING   Download failed: https://www.fshare.vn/file/YLTZ915BT1BA | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:09 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/YLTZ915BT1BA (pid:9)
10.04.2020 02:19:09 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:11 INFO      Download starts: https://www.fshare.vn/file/67886VOJDV4M
10.04.2020 02:19:11 WARNING   HOSTER FshareVn[75]: Plugin may be unstable
10.04.2020 02:19:11 WARNING   Download failed: https://www.fshare.vn/file/67886VOJDV4M | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:11 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/67886VOJDV4M (pid:9)
10.04.2020 02:19:11 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:13 INFO      Download starts: https://www.fshare.vn/file/5Y4DA1ZMWCQC
10.04.2020 02:19:13 WARNING   HOSTER FshareVn[76]: Plugin may be unstable
10.04.2020 02:19:13 WARNING   Download failed: https://www.fshare.vn/file/5Y4DA1ZMWCQC | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:13 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/5Y4DA1ZMWCQC (pid:9)
10.04.2020 02:19:13 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:15 INFO      Download starts: https://www.fshare.vn/file/7CUXVBPA259L
10.04.2020 02:19:15 WARNING   HOSTER FshareVn[77]: Plugin may be unstable
10.04.2020 02:19:15 WARNING   Download failed: https://www.fshare.vn/file/7CUXVBPA259L | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:15 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/7CUXVBPA259L (pid:9)
10.04.2020 02:19:15 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:17 INFO      Download starts: https://www.fshare.vn/file/9CPAXO9ZXHBU
10.04.2020 02:19:17 WARNING   HOSTER FshareVn[78]: Plugin may be unstable
10.04.2020 02:19:17 WARNING   Download failed: https://www.fshare.vn/file/9CPAXO9ZXHBU | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:17 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/9CPAXO9ZXHBU (pid:9)
10.04.2020 02:19:17 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:19 INFO      Download starts: https://www.fshare.vn/file/51EP32OYIE8O
10.04.2020 02:19:19 WARNING   HOSTER FshareVn[79]: Plugin may be unstable
10.04.2020 02:19:19 WARNING   Download failed: https://www.fshare.vn/file/51EP32OYIE8O | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:19 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/51EP32OYIE8O (pid:9)
10.04.2020 02:19:19 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:21 INFO      Download starts: https://www.fshare.vn/file/ZWAF2P33TMET
10.04.2020 02:19:21 WARNING   HOSTER FshareVn[80]: Plugin may be unstable
10.04.2020 02:19:21 WARNING   Download failed: https://www.fshare.vn/file/ZWAF2P33TMET | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:21 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/ZWAF2P33TMET (pid:9)
10.04.2020 02:19:21 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:23 INFO      Download starts: https://www.fshare.vn/file/DZMV9DTOP7NE
10.04.2020 02:19:23 WARNING   HOSTER FshareVn[81]: Plugin may be unstable
10.04.2020 02:19:23 WARNING   Download failed: https://www.fshare.vn/file/DZMV9DTOP7NE | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:23 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/DZMV9DTOP7NE (pid:9)
10.04.2020 02:19:23 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:25 INFO      Download starts: https://www.fshare.vn/file/CTUHK5S8F8KY
10.04.2020 02:19:25 WARNING   HOSTER FshareVn[82]: Plugin may be unstable
10.04.2020 02:19:25 WARNING   Download failed: https://www.fshare.vn/file/CTUHK5S8F8KY | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:25 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/CTUHK5S8F8KY (pid:9)
10.04.2020 02:19:25 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:27 INFO      Download starts: https://www.fshare.vn/file/LF1KIMBDP7L2
10.04.2020 02:19:27 WARNING   HOSTER FshareVn[83]: Plugin may be unstable
10.04.2020 02:19:27 WARNING   Download failed: https://www.fshare.vn/file/LF1KIMBDP7L2 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:27 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/LF1KIMBDP7L2 (pid:9)
10.04.2020 02:19:27 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:29 INFO      Download starts: https://www.fshare.vn/file/33YDKGEAYROU
10.04.2020 02:19:29 WARNING   HOSTER FshareVn[84]: Plugin may be unstable
10.04.2020 02:19:29 WARNING   Download failed: https://www.fshare.vn/file/33YDKGEAYROU | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:29 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/33YDKGEAYROU (pid:9)
10.04.2020 02:19:29 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:31 INFO      Download starts: https://www.fshare.vn/file/QSJ19YQJ6Y26
10.04.2020 02:19:31 WARNING   HOSTER FshareVn[85]: Plugin may be unstable
10.04.2020 02:19:31 WARNING   Download failed: https://www.fshare.vn/file/QSJ19YQJ6Y26 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:31 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/QSJ19YQJ6Y26 (pid:9)
10.04.2020 02:19:31 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:33 INFO      Download starts: https://www.fshare.vn/file/G7GBYKRUNH4V
10.04.2020 02:19:33 WARNING   HOSTER FshareVn[86]: Plugin may be unstable
10.04.2020 02:19:33 WARNING   Download failed: https://www.fshare.vn/file/G7GBYKRUNH4V | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:33 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/G7GBYKRUNH4V (pid:9)
10.04.2020 02:19:33 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:35 INFO      Download starts: https://www.fshare.vn/file/BV6WS1GWXN7R
10.04.2020 02:19:35 WARNING   HOSTER FshareVn[87]: Plugin may be unstable
10.04.2020 02:19:35 WARNING   Download failed: https://www.fshare.vn/file/BV6WS1GWXN7R | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:35 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/BV6WS1GWXN7R (pid:9)
10.04.2020 02:19:35 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:37 INFO      Download starts: https://www.fshare.vn/file/CJGEB5GNLQNQ
10.04.2020 02:19:37 WARNING   HOSTER FshareVn[88]: Plugin may be unstable
10.04.2020 02:19:37 WARNING   Download failed: https://www.fshare.vn/file/CJGEB5GNLQNQ | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:37 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/CJGEB5GNLQNQ (pid:9)
10.04.2020 02:19:37 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:39 INFO      Download starts: https://www.fshare.vn/file/S4TL817M94I1
10.04.2020 02:19:39 WARNING   HOSTER FshareVn[89]: Plugin may be unstable
10.04.2020 02:19:39 WARNING   Download failed: https://www.fshare.vn/file/S4TL817M94I1 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:39 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/S4TL817M94I1 (pid:9)
10.04.2020 02:19:39 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:41 INFO      Download starts: https://www.fshare.vn/file/9QNROSMHCWBK
10.04.2020 02:19:41 WARNING   HOSTER FshareVn[90]: Plugin may be unstable
10.04.2020 02:19:41 WARNING   Download failed: https://www.fshare.vn/file/9QNROSMHCWBK | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:41 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/9QNROSMHCWBK (pid:9)
10.04.2020 02:19:41 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:43 INFO      Download starts: https://www.fshare.vn/file/Q7KXROXBICYG
10.04.2020 02:19:43 WARNING   HOSTER FshareVn[91]: Plugin may be unstable
10.04.2020 02:19:43 WARNING   Download failed: https://www.fshare.vn/file/Q7KXROXBICYG | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:43 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/Q7KXROXBICYG (pid:9)
10.04.2020 02:19:43 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:45 INFO      Download starts: https://www.fshare.vn/file/QSFGR58E1MU2
10.04.2020 02:19:45 WARNING   HOSTER FshareVn[92]: Plugin may be unstable
10.04.2020 02:19:45 WARNING   Download failed: https://www.fshare.vn/file/QSFGR58E1MU2 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:45 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/QSFGR58E1MU2 (pid:9)
10.04.2020 02:19:45 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:47 INFO      Download starts: https://www.fshare.vn/file/L11QGSBU3LG6
10.04.2020 02:19:47 WARNING   HOSTER FshareVn[93]: Plugin may be unstable
10.04.2020 02:19:47 WARNING   Download failed: https://www.fshare.vn/file/L11QGSBU3LG6 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:47 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/L11QGSBU3LG6 (pid:9)
10.04.2020 02:19:47 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:49 INFO      Download starts: https://www.fshare.vn/file/8DUY1YFX8XU7
10.04.2020 02:19:49 WARNING   HOSTER FshareVn[94]: Plugin may be unstable
10.04.2020 02:19:49 WARNING   Download failed: https://www.fshare.vn/file/8DUY1YFX8XU7 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:49 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/8DUY1YFX8XU7 (pid:9)
10.04.2020 02:19:49 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:51 INFO      Download starts: https://www.fshare.vn/file/3J75TGK6UHNI
10.04.2020 02:19:51 WARNING   HOSTER FshareVn[95]: Plugin may be unstable
10.04.2020 02:19:51 WARNING   Download failed: https://www.fshare.vn/file/3J75TGK6UHNI | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:51 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/3J75TGK6UHNI (pid:9)
10.04.2020 02:19:51 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:53 INFO      Download starts: https://www.fshare.vn/file/APGVPO2XBEQC
10.04.2020 02:19:53 WARNING   HOSTER FshareVn[96]: Plugin may be unstable
10.04.2020 02:19:53 WARNING   Download failed: https://www.fshare.vn/file/APGVPO2XBEQC | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:53 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/APGVPO2XBEQC (pid:9)
10.04.2020 02:19:53 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:55 INFO      Download starts: https://www.fshare.vn/file/L12V17NGY8WO
10.04.2020 02:19:55 WARNING   HOSTER FshareVn[97]: Plugin may be unstable
10.04.2020 02:19:55 WARNING   Download failed: https://www.fshare.vn/file/L12V17NGY8WO | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:55 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/L12V17NGY8WO (pid:9)
10.04.2020 02:19:55 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:57 INFO      Download starts: https://www.fshare.vn/file/IMFVWM7RY82A
10.04.2020 02:19:57 WARNING   HOSTER FshareVn[98]: Plugin may be unstable
10.04.2020 02:19:57 WARNING   Download failed: https://www.fshare.vn/file/IMFVWM7RY82A | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:57 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/IMFVWM7RY82A (pid:9)
10.04.2020 02:19:57 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:19:59 INFO      Download starts: https://www.fshare.vn/file/FSD9WHD6IS4Y
10.04.2020 02:19:59 WARNING   HOSTER FshareVn[99]: Plugin may be unstable
10.04.2020 02:19:59 WARNING   Download failed: https://www.fshare.vn/file/FSD9WHD6IS4Y | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:19:59 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/FSD9WHD6IS4Y (pid:9)
10.04.2020 02:19:59 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:20:01 INFO      Download starts: https://www.fshare.vn/file/1ZO4AQKF3X4G
10.04.2020 02:20:01 WARNING   HOSTER FshareVn[100]: Plugin may be unstable
10.04.2020 02:20:01 WARNING   Download failed: https://www.fshare.vn/file/1ZO4AQKF3X4G | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:20:01 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/1ZO4AQKF3X4G (pid:9)
10.04.2020 02:20:01 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:20:03 INFO      Download starts: https://www.fshare.vn/file/X97VNDJDYFZE
10.04.2020 02:20:03 WARNING   HOSTER FshareVn[101]: Plugin may be unstable
10.04.2020 02:20:03 WARNING   Download failed: https://www.fshare.vn/file/X97VNDJDYFZE | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:20:03 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/X97VNDJDYFZE (pid:9)
10.04.2020 02:20:03 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:20:05 INFO      Download starts: https://www.fshare.vn/file/I19XN9ZYKM92
10.04.2020 02:20:05 WARNING   HOSTER FshareVn[102]: Plugin may be unstable
10.04.2020 02:20:05 WARNING   Download failed: https://www.fshare.vn/file/I19XN9ZYKM92 | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:20:05 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/I19XN9ZYKM92 (pid:9)
10.04.2020 02:20:05 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:20:07 INFO      Download starts: https://www.fshare.vn/file/QVYU4OTVL3EH
10.04.2020 02:20:07 WARNING   HOSTER FshareVn[103]: Plugin may be unstable
10.04.2020 02:20:07 WARNING   Download failed: https://www.fshare.vn/file/QVYU4OTVL3EH | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:20:07 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/QVYU4OTVL3EH (pid:9)
10.04.2020 02:20:07 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 02:20:09 INFO      Download starts: https://www.fshare.vn/file/LKOH5ZA5LACJ
10.04.2020 02:20:09 WARNING   HOSTER FshareVn[104]: Plugin may be unstable
10.04.2020 02:20:09 WARNING   Download failed: https://www.fshare.vn/file/LKOH5ZA5LACJ | 'NoneType' object has no attribute 'get_info'
10.04.2020 02:20:09 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://www.fshare.vn/file/LKOH5ZA5LACJ (pid:9)
10.04.2020 02:20:09 INFO      ADDON UnSkipOnFail: No duplicates found
10.04.2020 04:37:56 INFO      ADDON UpdateManager: pyLoad is up to date!
10.04.2020 04:37:56 INFO      ADDON UpdateManager: All plugins are up to date!

```

**Comment by GammaC0de** [comment]
Please do not clear the browser  account history

**Comment by bigdargon** [comment]
and the option to not download with free didn't work, I tried turning it off but pyload still waited 1 minute and downloaded.  The free download speed is very slow while my account is paid.  Thanks so much for the update!
